resources/analysisTools/copyNumberEstimationWorkflow/phasing.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Unfortunately, I cannot make such a deep review to check all this :-(

Reply:
The integrated test from the Beagle version provides almost identical CNVs as impute2 version for hg19, so we hope it works as intended :)